sirbot-pyslackers/endpoints/apscheduler.py

```diff
@@ -57,3 +61,31 @@ def create_jobs(scheduler, bot):
                 user,
             )
     LOG.info("List of slack users up to date")
+
+
+async def etc_finance_bell_open(bot):
```

Comment:
I wonder if these should be imported from a dedicated module, it seems like the apscheduler one should really just be defining the schedule and registering the functions, but the functions should be logically grouped. Of course, I recognize the example here is all in one file